oml/metrics/embeddings.py

```diff
@@ -66,6 +66,7 @@ def __init__(
         cmc_top_k: Tuple[int, ...] = (5,),
         precision_top_k: Tuple[int, ...] = (5,),
         map_top_k: Tuple[int, ...] = (5,),
+        fmr_vals: Tuple[float, ...] = (0.1,),
```

Comment:
would `fmr_quantiles` be a better name for this argument?

Reply:
Yeah, agree. And I think that maybe it will be better to make it not a float number in range [0, 1], but an int number in range [0, 100], like a percentile?

Reply:
I do agree with the idea of using [0, 100] range